### DISCUSSION TITLE
Sort PRs by merge date

### DISCUSSION THREAD

**Discussion by Popov72** [community/community#60876] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,

I think it would be interesting to be able to sort PRs by merge date.

There have been several times when I wanted to refer to a PR that I knew had been merged recently but I couldn't remember a keyword from the author's title/pseudo, and sorting the closed PRs by merge date made that PR appear on the 4th page because it had been created several months ago.

**Comment by jlmobbex** [comment]
https://github.com/isaacs/github/issues/1600

**Reply by johnnybaloney** [reply]
>  This repository has been archived by the owner on Nov 18, 2021. It is now read-only. 

[The script](https://github.com/isaacs/github/issues/1600#issuecomment-673467801) still works, though.

For the dates to display I had to update the other script to:

```
var times = document.getElementsByTagName("relative-time");
for(var i=0; i<times.length; i++) {
    var datetime = times[i].getAttribute("datetime");
    var shadow = times[i].shadowRoot;
    shadow.innerHTML = datetime.replace("T", " ").replace("Z", " ");
};
```
